src/main/java/com/es/boardGameTraining/dto/GameDTO.java

```diff
@@ -0,0 +1,107 @@
+package com.es.boardGameTraining.dto;
+
+public class GameDTO {
+    private Long id;
+
+    private String title;
+
+    private String author;
+
+    private String artist;
```

Comment:
Please, review the fields in [this PR](https://github.com/xebia-functional/board-game-training/pull/24)